$MirOS: ports/www/lynx/patches/patch-userdefs_h,v 1.13 2009/08/09 15:46:33 tg Exp $

	Upstream (Lynx) should at least enable KANJI_CODE_OVERRIDE by
	default and incorporate the #ifndef LYNX_VERSION part.
	Setting PREFERRED_LANGUAGE to "" is a sensible thing to do too.

--- userdefs.h.orig	Sat Dec 11 15:26:08 2010
+++ userdefs.h	Thu Dec 23 20:13:40 2010
@@ -339,7 +339,7 @@
  * use any default viewers for image types.  Note that open is used as
  * the default for NeXT, instead of the XLOADIMAGE_COMMAND definition.
  */
-#define XLOADIMAGE_COMMAND "xli %s &"
+#define XLOADIMAGE_COMMAND "xloadimage -quiet %s"
 
 /**************************
  * For UNIX systems, SYSTEM_MAIL and SYSTEM_MAIL_FLAGS are set by the
@@ -473,7 +473,7 @@
  * Normally we expect you will connect to a remote site, e.g., the Lynx starting
  * site:
  */
-#define STARTFILE "http://lynx.isc.org/"
+/* #define STARTFILE "http://lynx.isc.org/" */
 /*
  * As an alternative, you may want to use a local URL.  A good choice for this
  * is the user's home directory:
@@ -483,6 +483,7 @@
  * you can connect to reliably.  Otherwise users will become confused and think
  * that they cannot run Lynx.
  */
+#define STARTFILE "file://localhost/~/"
 
 /*****************************
  * HELPFILE must be defined as a URL and must have a
@@ -739,7 +740,7 @@
  * that preference in their RC file.  This may be a comma-separated list
  * of languages in decreasing preference.
  */
-#define PREFERRED_LANGUAGE "en"
+#define PREFERRED_LANGUAGE ""
 
 /*****************************
  * PREFERRED_CHARSET specifies the character set in MIME notation (e.g.,
@@ -814,8 +815,8 @@
  * and use "http://" as the default (e.g., gopher.wfbr.edu or gopher.wfbr.
  * will be made gopher://gopher.wfbr.edu).
  */
-#define URL_DOMAIN_PREFIXES "www."
-#define URL_DOMAIN_SUFFIXES ".com,.edu,.net,.org"
+#define URL_DOMAIN_PREFIXES ""
+#define URL_DOMAIN_SUFFIXES ""
 
 /********************************
  * If LIST_NEWS_NUMBERS is set TRUE, Lynx will use an ordered list
@@ -856,7 +857,7 @@
  * directory.  If it is in a subdirectory, begin it with a dot-slash
  * (e.g., ./lynx/.lynxsig).  The definition here can be changed in lynx.cfg.
  */
-#define LYNX_SIG_FILE ".lynxsig"
+#define LYNX_SIG_FILE ".etc/sig"
 
 /********************************
  * BIBP_GLOBAL_SERVER is the default global server for bibp: links, used
@@ -1432,7 +1433,9 @@
  * the version definition with the Project Version on checkout.  Just
  * ignore it. - kw */
 /* $Format: "#define LYNX_VERSION \"$ProjectVersion$\""$ */
+#ifndef LYNX_VERSION
 #define LYNX_VERSION "2.8.8dev.7"
+#endif
 #define LYNX_WWW_HOME "http://lynx.isc.org/"
 #define LYNX_WWW_DIST "http://lynx.isc.org/current/"
 /* $Format: "#define LYNX_DATE \"$ProjectDate$\""$ */
@@ -1467,7 +1470,7 @@
 #define FNAMES_8_3
 #endif
 
-#ifdef FNAMES_8_3
+#if 1 /* #ifdef FNAMES_8_3 */
 #define HTML_SUFFIX ".htm"
 #else
 #define HTML_SUFFIX ".html"
@@ -1477,7 +1480,7 @@
 #ifdef FNAMES_8_3
 #define FNAME_LYNXRC "lynx.rc"
 #else
-#define FNAME_LYNXRC ".lynxrc"
+#define FNAME_LYNXRC ".etc/lynxrc"
 #endif /* FNAMES_8_3 */
 #endif
 
@@ -1485,7 +1488,7 @@
 #ifdef FNAMES_8_3
 #define FNAME_LYNX_COOKIES "cookies"
 #else
-#define FNAME_LYNX_COOKIES ".lynx_cookies"
+#define FNAME_LYNX_COOKIES ".etc/cookies.txt"
 #endif /* FNAMES_8_3 */
 #endif
 
@@ -1651,11 +1654,11 @@
 
 /*****************************
  * Uncomment the following line to enable the kanji code override routine.
- * The code can be changed by pressing ^L.  More precisely, this allows
+ * The code can be changed by pressing Shift-J.  More precisely, this allows
  * the user to override the assumption about the kanji code for the document
  * which Lynx has made on the basis of a META tag and HTTP response.
  */
-/*#define KANJI_CODE_OVERRIDE */
+#define KANJI_CODE_OVERRIDE
 
 /**************************
  * SSL_CERT_FILE contains valid SSL CA certificates.  Set this to a string
